docs/source/alerts.rst

```diff
@@ -2605,3 +2606,30 @@ Example usage::
     zbx_key: "sender_load1"
 
 where ``hostname`` is the available elasticsearch field.
+
+YZJ
+~~~~~~~
+
+YZJ will send notification to a YZJ application. The body of the notification is formatted the same as with other alerters.
+
+Required:
+
+``yzj_token``:  The request token.
+
+Optional:
+
+``yzj_webhook_url``:  The webhook URL.
+
+``yzj_type``: Default 0, send text message. https://www.yunzhijia.com/opendocs/docs.html#/server-api/im/index?id=%e7%be%a4%e7%bb%84%e6%9c%ba%e5%99%a8%e4%ba%ba
+
+``yzj_proxy``: By default ElastAlert 2 will not use a network proxy to send notifications to YZJ. Set this option using ``hostname:port`` if you need to use a proxy. only supports https.
+
+``yzj_custom_loc``: The YZJ custom loc for privately deployed.
```

Comment:
This parameter's documentation needs more clarity. Is custom loc a hostname? port? protocol? Example would be helpful.

Reply:
It's `hostname:port`, like `www.xxx.com:8080`

Reply:
Please update the docs with that example.